jar-infer/build.gradle

```diff
@@ -23,19 +23,9 @@ repositories {
 }
 
 dependencies {
-    compile group: 'com.ibm.wala', name: 'com.ibm.wala.shrike', version: '1.4.3'
-    compile group: 'com.ibm.wala', name: 'com.ibm.wala.util', version: '1.4.3'
-    compile group: 'com.ibm.wala', name: 'com.ibm.wala.core', version: '1.4.3'
-    compile group: 'com.ibm.wala', name: 'com.ibm.wala.cast', version: '1.4.3'
-    compile group: 'com.ibm.wala', name: 'com.ibm.wala.cast.js', version: '1.4.3'
-    compile group: 'com.ibm.wala', name: 'com.ibm.wala.cast.js.rhino', version: '1.4.3'
-    compile group: 'com.ibm.wala', name: 'com.ibm.wala.cast.java', version: '1.4.3'
-    compile group: 'com.ibm.wala', name: 'com.ibm.wala.cast.java.ecj', version: '1.4.3'
-    // uncomment to use wala.dalvik or wala.scandroid
-    //compile group: 'com.ibm.wala', name: 'com.ibm.wala.dalvik', version: '1.4.3'
-    //compile group: 'com.ibm.wala', name: 'com.ibm.wala.scandroid', version: '1.4.3'
-
+    compile deps.build.wala
     compile deps.build.guava
+    compile deps.build.commonsIO
```

Comment:
Just curious, what is this?

Reply:
Previously the jar-infer code was depending on this but via a transitive dependence from some other WALA project; I made the dependence explicit.  You'll have to ask @subarnob what we use it for 😄